pages/instructorEffectiveUser/instructorEffectiveUser.js

```diff
@@ -43,7 +44,20 @@ router.get('/', function (req, res, next) {
     if (res.locals.ipaddress.substr(0, 7) === '::ffff:') {
       res.locals.ipaddress = res.locals.ipaddress.substr(7);
     }
-    res.locals.req_date_for_display = moment(res.locals.req_date).toISOString(true);
+    res.locals.true_req_date_for_display = format(
+      utcToZonedTime(res.locals.true_req_date, res.locals.course_instance.display_timezone),
```

Comment:
Was about to ask if we should be coalescing with `course.display_timezone`, but then I remembered that you discovered in https://github.com/PrairieLearn/PrairieLearn/issues/7344 that `course_instance_.display_timezone` is effectively non-null!